resources/lang/de/Template.properties

```
@@ -847,3 +847,6 @@ paginationLastPage = "Zur letzten Seite"
 ; Language Detection
 languageDetectionText = "Klicken Sie auf die Schaltfläche, um deutsche Inhalte zu sehen."
 languageDetectionButton = "Deutsche Website öffnen"
+; Language Detection
+: Aria Label
+closeIcon = "Schließen"
```

```suggestion
; Common
closeIcon = "Schließen"
```

This would be more accurate if you are using it at many points in the shop